semgrep/edgedict.yml

```diff
@@ -0,0 +1,49 @@
+rules:
```

Comment:
Might be good to include some docs or a makefile rule or something to show how to use these.  Or include them in CI.

Reply:
Oh right.  I'll include a README explaining that these are.